.travis.yml

```diff
@@ -123,7 +123,15 @@ matrix:
       env: BUILDTYPE=debug
       node_js: 4
       # Overrides `install` to avoid initializing clang toolchain
-      install: 
-        - ./scripts/format.sh
+      install:
+        # First run the clang-tidy target
+        # This target does not currently return non-zero when problems
+        # are fixed, but rather it fixes automatically. Changed files
+        # will be detected by the format target and will trigger an error
```

Comment:
Working around the lack of non-zero error code on error mentioned at https://github.com/mapbox/node-cpp-skel/pull/64/files#r133848127 here by running `make tidy` first and depending on `make format` to throw if it sees any changes. It would be ideal to error immediately in `make tidy` but I can't figure out how to get that working...